neuraxle/steps/util.py

```diff
@@ -196,17 +202,28 @@ def get_name_tape(self) -> List[str]:
 
 class StepClonerForEachDataInput(MetaStepMixin, BaseStep):
     def __init__(self, wrapped: BaseStep, copy_op=copy.deepcopy):
-        # TODO: set params on wrapped.
-        # TODO: use MetaStep*s*Mixin (plural) and review.
         BaseStep.__init__(self)
-        MetaStepMixin.__init__(self)
+        MetaStepMixin.__init__(self, wrapped)
```

Comment:
For Issue #29, it would be a bit safer to add this too in the set_hyperparams:
self.steps = [s.set_hyperparams(hyperparams.to_flat()) for s in self.steps]

This is to do for this PR, despite the diff for this seems to be in PR #68.

Reply:
@guillaume-chevalier done, I think we can merge this now